### PR TITLE
5164 4a submit def stats

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: web-client/src/presenter/actions/submitAddDeficiencyStatisticsAction.test.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import { submitAddDeficiencyStatisticsAction } from './submitAddDeficiencyStatisticsAction';
+
+presenter.providers.applicationContext = applicationContext;
+presenter.providers.path = {
+  error: jest.fn(),
+  success: jest.fn(),
+};
+
+describe('submitAddDeficiencyStatisticsAction', () => {
+  it('calls the other statistics submit interactor, returning the success path', async () => {
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/submitAddDeficiencyStatisticsAction.test.js]: Nit: test description needs to be updated

File: web-client/src/presenter/sequences/submitAddDeficiencyStatisticsSequence.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+export const submitAddDeficiencyStatisticsSequence = [
+  showProgressSequenceDecorator([
+    submitAddDeficiencyStatisticsAction,
+    {
+      error: [],
+      success: [
+        clearFormAction,
+        setSaveAlertsForNavigationAction,
+        setCaseDetailPageTabFrozenAction,
```
Comment (rachaelparris) [inline web-client/src/presenter/sequences/submitAddDeficiencyStatisticsSequence.js:18]: What does this "frozen" tab do?

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/presenter/sequences/submitAddDeficiencyStatisticsSequence.js (hunk 2 above)
Comment (matthopson) [reply web-client/src/presenter/sequences/submitAddDeficiencyStatisticsSequence.js:18]: Prevents from going to the default tab ('overview')

Comment (matthopson) [reply web-client/src/presenter/sequences/submitAddDeficiencyStatisticsSequence.js:18]: https://github.com/flexion/ef-cms/blob/develop/web-client/src/presenter/actions/setDefaultCaseDetailTabAction.js

Comment (rachaelparris) [reply web-client/src/presenter/sequences/submitAddDeficiencyStatisticsSequence.js:18]: Anyone else feel like the case detail tabs are getting a little complicated and could use a refactor?

Comment (matthopson) [reply web-client/src/presenter/sequences/submitAddDeficiencyStatisticsSequence.js:18]: Yeah. It'd be great to include them in the route for deep linking.

**Pull request merged**